### PR TITLE
New version: DeepL.DeepL version 23.11.2.11449

### LINKED ISSUE #131871: [Update Request]: DeepL
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

DeepL.DeepL

### Package Version

23.9.1.10740

### Please describe the changes you would like to see

It is not possible to update DeepL because it is throwing "The hash value of the installer does not match." error.

```
winget output:
winget update --all --include-unknown
Name      ID                                 Version   Verfügbar    Quelle
--------------------------------------------------------------------------
DeepL     DeepL.DeepL                        Unknown   23.9.1.10740 winget
UltraEdit IDMComputerSolutions,Inc.UltraEdit 30.2.0.27 30.2.27      winget
3 Aktualisierungen verfügbar.

Für die folgenden Pakete ist ein Upgrade verfügbar, für das Upgrade ist jedoch eine explizite Zielgruppenadressierung erforderlich:
Name    ID              Version  Verfügbar Quelle
-------------------------------------------------
Discord Discord.Discord 1.0.9018 1.0.9025  winget

Abhängigkeiten werden installiert:
(1/1) Gefunden DeepL [DeepL.DeepL] Version 23.9.1.10740
Diese Anwendung wird von ihrem Besitzer an Sie lizenziert.
Microsoft ist nicht verantwortlich und erteilt keine Lizenzen für Pakete von Drittanbietern.
Download läuft https://appdownload.deepl.com/windows/0install/DeepLSetup.exe
  ██████████████████████████████  4.28 MB / 4.28 MB
Der Hash-Wert des Installers stimmt nicht überein.

1 Pakete sind angeheftet und müssen explizit aktualisiert werden.
```

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve #131871

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/131887)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Dragon1573, It seem you have Need-Author-Feedback on your pr.

File: manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+InstallerSwitches:
+  Silent: -verysilent
+  SilentWithProgress: -silent
+  InstallLocation: --store-path="<INSTALLPATH>"
+UpgradeBehavior: install
+Protocols:
+- deepl
+ProductCode: https%3a##appdownload.deepl.com#windows#0install#deepl.xml
+Installers:
+- Architecture: x86
```
Comment (Dragon1573) [inline manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml:20]:
```suggestion
- Architecture: x64
```

File: manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  Scope: user
+  InstallerUrl: https://appdownload.deepl.com/windows/0install/DeepLSetup.exe
+  InstallerSha256: CAC980C207B8653C98A3FB671B2BF1FF59E49E8B36BB74A757935129A6F870C8
+- Architecture: x86
```
Comment (Dragon1573) [inline manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml:24]:
```suggestion
- Architecture: x64
```

**Review by Dragon1573 — COMMENTED** [review]
Last version `v23.9.1.10740` (#121259) shows the file is a 64-bit installer.

**Review by stephengillie — APPROVED** [review]

File: manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: -verysilent
+  SilentWithProgress: -silent
+  InstallLocation: --store-path="<INSTALLPATH>"
+UpgradeBehavior: install
+Protocols:
+- deepl
+ProductCode: https%3a##appdownload.deepl.com#windows#0install#deepl.xml
```
Comment (stephengillie) [inline manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml:18]: I made an error in approving this - should this be a valid URL?

Comment (Dragon1573) [reply manifests/d/DeepL/DeepL/23.11.2.11449/DeepL.DeepL.installer.yaml:18]: This product code is inspected by `wingetcreate.exe` and written to the manifest as is.

**Pull request merged**